provider/provider.go

```diff
@@ -143,6 +143,21 @@ func (a *openstackProvider) CreateInstance(ctx context.Context, bootstrapParams
 	if err != nil {
 		return params.ProviderInstance{}, fmt.Errorf("failed to resolve image info: %w", err)
 	}
+
+	// verify owner
+	if spec.AllowedImageOwners != nil && len(spec.AllowedImageOwners) > 0 {
```

Comment:
I think we can drop the nil check here. The nil value of a slice is an empty slice, even if we explicitly set it to nil. So `len()` should be fine.

Reply:
done